crates/sui/src/sui_commands.rs

```diff
@@ -400,15 +400,17 @@ async fn prompt_if_no_config(wallet_conf_path: &Path) -> Result<(), anyhow::Erro
                 .unwrap_or(&sui_config_dir()?)
                 .join(SUI_KEYSTORE_FILENAME);
             let keystore = KeystoreType::File(keystore_path);
-            println!("Generating keypair ...\n");
-
-            println!("Do you want to generate a secp256k1 keypair instead? [y/N] No will select Ed25519 by default. ");
+            println!("Select key scheme (0 for ed25519, 1 for secp256k1) to generate keypair:\n");
```

Comment:
This comment is not aligned with what 0 and 1 represent below: i.e. `0 for ed25519` but then
`"0" => SignatureScheme::Secp256k1`